web/src/filter.ts

```diff
@@ -71,17 +69,8 @@ type Part =
           prefix_for_operator: string;
           operand: string;
           is_empty_string_topic?: boolean;
-      }
-    | {
-          type: "user_pill";
-          operator: string;
-          users: ValidOrInvalidUser[];
```

Comment:
I think this commit would benefit from being renamed to something that called out it's only changing things for user suggestions. Also in the commit description: "we just need the description_html now" -- do you mean just the search_string?